docs/ref/models/fields.txt

```diff
@@ -2266,6 +2201,113 @@ accepted by :class:`ForeignKey`, plus one extra argument:
 See :doc:`One-to-one relationships </topics/db/examples/one_to_one>` for usage
 examples of ``OneToOneField``.
 
+.. _lazy-relationships:
+
+Lazy relationships
+------------------
+
+Lazy relationships allow you to reference models using their names as strings
+or to create recursive relationships. You can use strings as the first
+argument in any relationship field to reference models lazily. The following
+types of lazy references are available:
+
+.. _recursive-relationships:
+
+Recursive
+~~~~~~~~~
+
+To define a relationship where a model references itself, use ``"self"`` as
+the first argument of the relationship field::
+
+    from django.db import models
+
+
+    class Manufacturer(models.Model):
+        suppliers = models.ManyToManyField("self", symmetrical=False)
+
+When used in an :ref:`abstract model <abstract-base-classes>`, the recursive
+relationship will point to concrete model subclasses.
+
+Relative
+~~~~~~~~
+
+If you need to create a relationship on a model that has not yet been defined,
+you can use the name of the model instead of the model object itself::
+
+    from django.db import models
+
+
+    class Car(models.Model):
+        manufacturer = models.ForeignKey(
+            "Manufacturer",
+            on_delete=models.CASCADE,
+        )
+        # ...
+
+
+    class Manufacturer(models.Model):
+        # ...
+        pass
```

Comment:
I know this comes from the old content, but I would advice to use the same class definition as the one in `Recursive`, to avoid confusing the reader.